src/page-modules/assistant/details/utils.ts

```diff
@@ -1,19 +1,23 @@
 import { fromCETToLocalTime } from '@atb/utils/date';
 import { parseTripQueryString } from '../server/journey-planner';
+import { TranslateFunction } from '@atb/translations';
+import dictionary from '@atb/translations/dictionary';
 
-export function formatQuayName(quayName?: string, publicCode?: string | null) {
+export function formatQuayName(t: TranslateFunction, quayName?: string, publicCode?: string | null) {
   if (!quayName) return;
   if (!publicCode) return quayName;
-  return `${quayName} ${publicCode}`;
+  const prefix = t(dictionary.quayPublicCodePrefix);
```

Comment:
- Should the prefix be capitalized or not?
- Should there be a comma before the prefix?

I do not have any clear answer to these, but we should maybe consider them at least.